routes/auth-routes.js

```diff
@@ -1,140 +0,0 @@
-// The routes file will be defined in the routes/auth.routes.js, and we will set the necessary packages and code to signup in the application:
-
-const { Router } = require("express");
-const router = new Router();
-const passport = require("passport");
-const User = require("../models/user");
-const bcrypt = require("bcrypt");
-const bcryptSalt = 10;
-const { isLoggedIn, isLoggedOut } = require("../middleware/route-guard");
-const filteredDomains = require("../public/javascripts/filteredDomains");
-const nodemailer = require("nodemailer");
-const nodemailerSetup = require('../public/javascripts/nodemailer');
-
-// SIGNUP ROUTES, implemented with server-side validation on the email and password used. Users are also issued a verification code over email to verify their sign up.
-
-router.get("/signup", isLoggedOut, (req, res, next) => res.render("auth/signup", { layout: false }));
```

Comment:
You're not using 2 / 3 parameters passed here, would recommend using `_` and removing `next` like below instead:

```
router.get("/signup", isLoggedOut, (_, res) => res.render("auth/signup", { layout: false }));